Hearthstone Deck Tracker/Controls/Overlay/BattlegroundsMinions.xaml.cs

```diff
@@ -72,13 +78,17 @@ private void Update(int tier, IEnumerable<Race> availableRaces)
 				for(var i = 0; i < 6; i++)
 					_tierIcons[i].SetFaded(false);
 				Groups.Clear();
+				UnavailableTypes.UnavailableTypesVisibility = System.Windows.Visibility.Collapsed;
 				return;
 			}
 			for(var i = 0; i < 6; i++)
 				_tierIcons[i].SetFaded(i != tier - 1);
 
 			var resort = false;
-			
+
+			var unavailableRaces = string.Join(", ", _db.Value.Races.Where(x => !availableRaces.Contains(x) && x != Race.INVALID && x != Race.ALL).Select(x => HearthDbConverter.RaceConverter(x)));
+			UpdateUnavailableRaces(unavailableRaces);
```

Comment:
You can move the `unavailableRaces` string generation into the `UpdateUnavailableRaces` call. Alternatively you could inline the method but abstracting it away is fine.